public/app/plugins/panel/alertGroups/composable_panelcfg.cue

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 Grafana Labs
```

Comment:
Based on new convention, starting from [here](https://github.com/grafana/grafana/pull/61928/files), I think this file should be named just `panelcfg.cue`.

@sdboyer could we somehow force this kind of naming convention? (e.g. making the jenny -gen.go- to only read those; and/or display a warning)

Reply:
good catch! Should be fixed now.

Reply:
Excellent!